src/components/Card.js

```diff
@@ -16,7 +16,19 @@ export default class Card extends Component {
 
     this.state = {
       showCard: false,
-      cardType: ["Bar", "Show", "Queen"]
+      cardType: ["Bar", "Show", "Queen"],
+      icons: {
```

Comment:
I think the object of icons is super helpful for readability, but it's looking a little cluttered in state. Maybe this object could live in the helper.js file and be imported because we will also need an array of show category images. These could also be stored locally in an images file if we like. I'm not sure what is preferred.